### DISCUSSION TITLE
[probes] why is it not possible to avoid using liveness and readiness probes?

### DISCUSSION THREAD

**Discussion by Epado** [mia-platform/community#113] [Q&A] [answered]
Sometimes it happens that services do not expose meaningfull liveness and readiness probes, but still a TCP probe is used, why?

**Comment by fredmaggiowski** [comment]
That is highly opinionated but it is a bad practice not to have them; however I agree that the fact that the console forces you to have them under the hood is not good either; we should improve this behaviour

**Reply by Epado** [reply]
i totally agree, could we take it into consideration ?

**Reply by fredmaggiowski** [reply]
@davidebianchi @albertotessarotto FYI

**Comment by javferrod** [comment]
Hi everyone! I found this discussion while searching for how to disable the liveness and readiness probes from the Console. Is it possible with the newer versions of the Console?

**Reply by fredmaggiowski** [reply]
Hi, this has not changed yet and there is no plan to do so; we could implement probes disabling if there is a necessity

**Reply by fredmaggiowski** [reply, marked as answer]
Since v12.0.2 it is possible to disable probes all together

https://docs.mia-platform.eu/docs/12.x/release-notes/v12.0.2#probe-configuration-improvement
